bucket/bazel-buildtools.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "4.2.5",
+    "description": "Developer tools for working with Google's bazel buildtool.",
+    "homepage": "https://github.com/bazelbuild/buildtools",
+    "license": "Apache-2.0",
+    "suggest": {
+        "bazel": "bazel"
+    },
+    "architecture": {
+        "64bit": {
+            "url": [
+                "https://github.com/bazelbuild/buildtools/releases/download/4.2.5/buildifier-windows-amd64.exe#/buildifier.exe",
+                "https://github.com/bazelbuild/buildtools/releases/download/4.2.5/buildozer-windows-amd64.exe#/buildozer.exe",
+                "https://github.com/bazelbuild/buildtools/releases/download/4.2.5/unused_deps-windows-amd64.exe#/unused_deps.exe"
+            ],
+            "hash": [
+                "4185a40d3154cacbe8b79f570b94e2c6f74fc9e317362b7d028c2e6c94edf9ba",
+                "2a9a7176cbd3b2f0ef989502128efbafd3b156ddabae93b9c979cd4017ffa300",
+                "77b092cd6aaaa1a57da3381bb7dcde3d9e5a8f7e74ff21a2b24ec722c765b9ed"
+            ]
+        }
+    },
+    "bin": [
+        "buildifier",
+        "buildozer",
+        "unused_deps"
```

Comment:
You forgot the .exe extension for these.

Reply:
Modified